### ISSUE TITLE
Listen on link generated not called

### ISSUE THREAD

**Issue by thaitanloi365** [blackmenthor/flutter-branch-io#24]
Hi @blackmenthor !
in Android platform:
This function `FlutterBranchIoPlugin.listenToGeneratedLinkStream().listen((link) {});` never called unless App onResume.

When I debugged on Android, I found that this function `buo.generateshorturl` was not called, cause link was not generated.

in iOS, It's work perfectly

Please look and help to resolve it.

**Comment by blackmenthor** [comment]
Hi!

I guess the app's `onResume` will always be called right after `onCreate` right? so it'll always be called anyway, or I'm missing something here?

**Comment by karansingla-dev** [comment]
@blackmenthor  yeah you are right, but generatedLink listener is not listen the genarate link

**Comment by RodrigoSMarques** [comment]
Hi @singlakaran 
I believe that this plugin is abandoned. There are many issues waiting to be answered.
See this here: https://pub.dev/packages/flutter_branch_sdk
